fix: accessibility issue in clear all cta (#1000)
react-tags/react-tags: README.md

|[`clearAll`](#clearAll) | `boolean` | `false` | Implies whether 'clear all' button should be shown.
|[`onClearAll`](#onClearAll) | `Function` |  | This callback if present is triggered when clear all button is clicked.
| [`maxTags`](#maxTags) | `number` | | The maximum count of tags to be added

> | [`ariaAttrs`](#ariaAttrs) | Object |`{clearAllLabel: "clear all tags"}` | An object used to pass custom accessibility attributes 

Reviewer: ```suggestion
| [`ariaAttrs`](#ariaAttrs) | Object |`{clearAllLabel: "clear all tags"}` | An object containing custom ARIA attributes to enhance screen reader support and improve accessibility
```

### maxTags
This prop specifies the maximum count of tags to be added. Incase the tags exceed, error will show up to convey the maximum tag limit has reached.

### ariaAttrs

> An object used to pass custom accessibility attributes (ARIA properties) to enhance screen reader support and improve accessibility. For example, clearAllLabel defines the accessible label for a "clear all tags" button, providing context for assistive technologies.

Reviewer: ```suggestion
An object containing custom ARIA attributes to enhance screen reader support and improve accessibility. 
It accepts the below attributes
```js
{
  clearAllLabel?: string // Optional accessible label for a "clear all tags" button improving accessibility by providing a clear, descriptive action label.
}
```
